modules/bridge/src/main/ts/ephox/bridge/api/Main.ts

```diff
@@ -7,9 +7,11 @@ import * as PublicInlineContent from './PublicInlineContent';
 import * as PublicMenu from './PublicMenu';
 import * as PublicSidebar from './PublicSidebar';
 import * as PublicToolbar from './PublicToolbar';
+import * as PublicView from './PublicView';
```

Comment:
Just as a quick note it looks like we need a bridge changelog entry for this?

Reply:
I added one for bridge now.